gen3-integration-tests/gen3_ci/jenkins_jobs/check-kube-pod.groovy

```diff
@@ -0,0 +1,97 @@
+/*
+    String parameter JOBNAME
```

Comment:
Is this job only to check job pods or any pod?

Reply:
This job is to only check the ephermal job pods. But if we want we can change this script a little to check the status of any pod in the namespace and see if they are completed if its a job pod or running if its a service pod

Reply:
let's name it more appropriately